public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -147,6 +130,28 @@ export class UnConnectedExploreToolbar extends PureComponent<Props, {}> {
     });
   };
 
+  stopLive = () => {
+    const { exploreId } = this.props;
+    // TODO referencing this from perspective of refresh picker when there is designated button for it now is not
```

Comment:
Not sure if it makes more sense here or other PR, but definitely don't won't to leave it like that.